src/helpers/utils.js

```diff
@@ -1,5 +1,8 @@
+/* eslint-disable valid-jsdoc */
 import bcrypt from 'bcrypt';
 import jwt from 'jsonwebtoken';
+import slug from 'slug';
+import crypto from 'crypto';
```

Comment:
Please remove crypto, you would not be needing it anymore

Reply:
Thank you for this review. This has been resolved.